dev/lint-python

```diff
@@ -47,11 +48,29 @@ if [ ! -e "$PEP8_SCRIPT_PATH" ]; then
     fi
 fi
 
+# Easy install pylint in /dev/pylint. To easy_install into a directory, the PYTHONPATH should
+# be set to the directory.
+# dev/pylint should be appended to the PATH variable as well.
+export "PYTHONPATH=$SPARK_ROOT_DIR/dev/pylint"
+export "PYLINT_HOME=$PYTHONPATH"
+export "PATH=$PYTHONPATH:$PATH"
+
+if [ ! -d "$PYLINT_HOME" ]; then
+    mkdir "$PYLINT_HOME"
+    easy_install -d "$PYLINT_HOME" pylint
```

Comment:
Maybe we should specify / freeze a specific PyLint version so that we don't start to get unexpected Jenkins failures if a new version is pushed to PyPi.